### PR TITLE
Clients MAY ignore challenges for any reason (but let's keep some examples)

### LINKED ISSUE #385: Ignoring challenges
From @paulwouters

> Clients can ignore the challenge if

Why "can" and not MAY ?

### PR DESCRIPTION
**Pull request by chris-wood**
Closes #385

### CODE REVIEW

**Review by tfpauly — APPROVED** [review]
Not too sure about "if desired", but I couldn't come up with a better suggestion.

File: draft-ietf-privacypass-auth-scheme.md [hunk 1]
```diff
@@ -219,8 +219,8 @@ using an out-of-band mechanism.
 - "max-age", an optional parameter that consists of the number of seconds for
 which the challenge will be accepted by the origin.
 
-Clients can ignore the challenge if the token-key is invalid or otherwise
-untrusted.
+Clients MAY ignore the challenge if desired, e.g., because the token-key is
```
Comment (chris-wood) [inline draft-ietf-privacypass-auth-scheme.md:222]:
```suggestion
Clients MAY ignore the challenge, e.g., because the token-key is
```

**Comment by chris-wood** [comment]
> Not too sure about "if desired", but I couldn't come up with a better suggestion.

Yeah, I wasn't too sure about it either. I just dropped it. 👍

**Comment by tfpauly** [comment]
Sold!

**Pull request merged**